shared/common-adapters/box.native.js

```diff
@@ -1,2 +1,71 @@
 // @flow
-export {View as default} from 'react-native'
+import * as React from 'react'
+import {View} from 'react-native'
+import {globalStyles, collapseStyles, globalMargins} from '../styles'
+import {intersperseFn} from '../util/arrays'
+
+import type {Box2Props} from './box'
+
+const Box = View
+
+const injectGaps = (Component, _children, gap, gapStart, gapEnd) => {
```

Comment:
worth pulling this out into `box.shared.js` or something?

Reply:
i figured this is so low level and it'll barely change it wasn't above the threshold to do that yet, could go either way